src/databaseAdapters/databaseInterfaces/user.ts

```diff
@@ -0,0 +1,20 @@
+import { IUser } from '../../gameplay/types';
+import { S3AvatarSet } from '../../clients/s3/S3Agent';
```

Comment:
Would be nice if this guy was in a types file and not dependent on the other stuff in this import, but not a big problem.